### PR TITLE
Add race and ethnicity to subjects

### LINKED ISSUE #361: Add additional subject fields for race and ethnicity
Subject entities should track race, ethnicity, and sex for reporting purposes.

Race field should have the following categories:  
- American Indian or Alaska Native  
- Asian  
- Native Hawaiian or Other Pacific Islander  
- Black or African American  
- White  
- More than One Race  
- Unknown or Not Reported

Ethnicity should have the following categories:  
- Not Hispanic or Latino  
- Hispanic or Latino  
- Unknown or Not Reported

*Sex is already tracked, but will need to be reported on based on Race and Ethnicity

### PR DESCRIPTION
**Pull request by nagem**
Closes #361

### CODE REVIEW

**Comment by rentzso** [comment]
LGTM

File: api/schemas/input/subject.json [hunk 1]
```diff
@@ -7,11 +7,15 @@
     "lastname":         { "type": "string", "maxLength": 64 },
     "firstname_hash":   { "type": ["string", "null"] },
     "lastname_hash":    { "type": ["string", "null"] },
+
+    "age":              { "type": ["number", "null"] },
     "sex":              { "enum": ["male", "female", "other", "unknown", null] },
+    "race":             { "enum": ["American Indian or Alaska Native", "Asian", "Native Hawaiian or Other Pacific Islander", "Black or African American", "White", "More Than One Race", "Unknown or Not Reported", null] },
+    "ethnicity":        { "enum": ["Not Hispanic or Latino", "Hispanic or Latino", "Unknown or Not Reported", null] },
```
Comment (kofalt) [inline api/schemas/input/subject.json]: Out of curiosity, what drives these fields? Homegrown? BIDS? Some other standard?

Comment (kofalt) [inline api/schemas/input/subject.json]: Also, sex is lower-case but race/ethnicity is title case? Intended?

Comment (nagem) [inline api/schemas/input/subject.json]: The PHS Inclusion Enrollment Report has these fields available.

**Pull request merged**